OpenSourcePolitics/decidim-module-blog_author_petition
# enables to manage blogs

#### :tophat: Description

This PR enables petition authors to manage their component blog.
They now can create update and destroy posts.

#### :pushpin: Related Issues
*Link your PR to an issue*

- [Notion card](https://www.notion.so/opensourcepolitics/bd4bf860eab94f9daca7da803f09e3b2?v=b816eedbd04643008c0d78f580a0e6a0&p=e4f77c43ff234c569185f1359ea720d2&pm=c)

#### Testing
* Log in as new user
* create petition
* Log out and log in as admin
* Create a blog and publish it in your petition
* Log out and log in as user
* visit blog page
* See you can create a new article
* Create it, and see you can update it
* Update it, and see you can destroy it
* Destroy it and see it's gone
* Log out and log in as different user
* Visit again the blog component
* See you cannot do this again
